dev/CommonStyles/FlipView_themeresources.xaml

```diff
@@ -85,8 +85,6 @@
     </ResourceDictionary.ThemeDictionaries>
 
     <x:Double x:Key="FlipViewButtonFontSize">8</x:Double>
-    <x:Double x:Key="FlipViewButtonFontSizePointerOver">9</x:Double>
-    <x:Double x:Key="FlipViewButtonFontSizePressed">7</x:Double>
```

Comment:
@ranjeshj @StephenLPeters want to make sure that we can remove these resources

Reply:
This is my only question with the change as well.

Reply:
Ranjesh says its fine :)